client/packages/common/src/ui/forms/JsonForms/components/text.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { ControlProps, rankWith, schemaTypeIs } from '@jsonforms/core';
+import { withJsonFormsControlProps } from '@jsonforms/react';
+import { DetailInputWithLabelRow } from '@openmsupply-client/common';
+
+export const stringTester = rankWith(3, schemaTypeIs('string'));
+
+const UIComponent = (props: ControlProps) => {
+  const { data, handleChange, label, path } = props;
+  return (
+    <DetailInputWithLabelRow
+      label={label}
+      inputProps={{
+        value: data,
+        sx: { margin: 0.5, width: '100%' },
+        disabled: false,
```

Comment:
isn't this the default value for `disabled`?

Reply:
Yes -- I think I put that in there as a reminder that I needed to update it with a dynamic value based on whether it's editable or now. I'll remove for now.